### PR TITLE
Edit authenticate-packages.md

### PR DESCRIPTION
**Pull request by johann1301s**
It is confusing as to why you would need a login token when accessing a public package. Shouldn't the pointer in the local .npmrc file be enough? I.e `@<scope>:registry=https://npm.pkg.github.com`.

This change should make things clearer. Notice from the two posts linked here show that I am not alone in finding the need for the token strange. 

"This is an public package.Is it should be auth too?" - from post.

Stack overflow issues:

https://stackoverflow.com/questions/73247706/unable-to-install-public-npm-package-from-github-package-registry-using-github-a?noredirect=1#comment129360034_73247706

https://stackoverflow.com/questions/60491581/installing-public-npm-packages-from-the-github-package-registry/73253496#73253496

----

Please feel free to modify how I changed the sentence, but make it explicit that the token regards both private AND public packages.

### CODE REVIEW

**Comment by cmwilson21** [comment]
@johann1301s Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by johann1301s** [comment]
Forgive me, but I asked a new question on stack overflow related to this:

https://stackoverflow.com/questions/73262134/why-are-personal-access-tokens-necessary-to-install-public-github-packages

**Review by tranvanlam2001 — COMMENTED** [review]
>

**Comment by aitemour79** [comment]
***@***.*** com

**Comment by vgrl** [comment]
Thanks @johann1301s ⚡, I'm going to make a small edit, and then I'll merge this.

File: data/reusables/package_registry/authenticate-packages.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-You need an access token to publish, install, and delete packages.
+You need an access token to publish, install, and delete both private and public packages.
```
Comment (vgrl) [inline data/reusables/package_registry/authenticate-packages.md:1]: Tiny change to include internal packages also.

```suggestion
You need an access token to publish, install, and delete private, internal, and public packages.
```

**Review by vgrl — DISMISSED** [review]
👍 after including internal packages.

**Review by vgrl — APPROVED** [review]
👍 Looks good in preview, for example in "[Introduction to GitHub Packages](https://docs-19552-c41b29.preview.ghdocs.com/en/packages/learn-github-packages/introduction-to-github-packages#authenticating-to-github-packages)".

**Pull request merged**